keyboards/keebio/quefrency/rev6/keyboard.json

```diff
@@ -0,0 +1,633 @@
+            {"x": 170, "y": 4, "flags": 2},
+            {"matrix": [5, 4], "x": 176, "y": 0, "flags": 4},
+            {"matrix": [5, 5], "x": 188, "y": 0, "flags": 4},
+            {"x": 194, "y": 4, "flags": 2},
+            {"matrix": [5, 6], "x": 200, "y": 0, "flags": 4},
+            {"matrix": [5, 7], "x": 212, "y": 0, "flags": 4},
+            {"x": 218, "y": 4, "flags": 2},
+            {"matrix": [5, 8], "x": 224, "y": 0, "flags": 4},
+
+            {"matrix": [6, 8], "x": 224, "y": 16, "flags": 4},
+            {"matrix": [6, 7], "x": 209, "y": 16, "flags": 4},
+            {"matrix": [6, 6], "x": 194, "y": 16, "flags": 4},
+            {"matrix": [6, 5], "x": 182, "y": 16, "flags": 4},
+            {"matrix": [6, 4], "x": 170, "y": 16, "flags": 4},
+            {"matrix": [6, 3], "x": 157, "y": 16, "flags": 4},
+            {"matrix": [6, 2], "x": 145, "y": 16, "flags": 4},
+            {"matrix": [6, 1], "x": 133, "y": 16, "flags": 4},
+            {"matrix": [6, 0], "x": 121, "y": 16, "flags": 4},
+
+            {"matrix": [7, 0], "x": 124, "y": 32, "flags": 4},
+            {"matrix": [7, 1], "x": 136, "y": 32, "flags": 4},
+            {"matrix": [7, 2], "x": 148, "y": 32, "flags": 4},
+            {"matrix": [7, 3], "x": 160, "y": 32, "flags": 4},
+            {"matrix": [7, 4], "x": 173, "y": 32, "flags": 4},
+            {"matrix": [7, 5], "x": 185, "y": 32, "flags": 4},
+            {"matrix": [7, 7], "x": 204, "y": 32, "flags": 4},
+            {"matrix": [7, 8], "x": 224, "y": 32, "flags": 4},
+
+            {"matrix": [8, 8], "x": 224, "y": 48, "flags": 4},
+            {"matrix": [8, 7], "x": 212, "y": 48, "flags": 4},
+            {"matrix": [8, 6], "x": 195, "y": 48, "flags": 4},
+            {"matrix": [8, 4], "x": 179, "y": 48, "flags": 4},
+            {"matrix": [8, 3], "x": 166, "y": 48, "flags": 4},
+            {"matrix": [8, 2], "x": 154, "y": 48, "flags": 4},
+            {"matrix": [8, 1], "x": 142, "y": 48, "flags": 4},
+            {"x": 138, "y": 52, "flags": 2},
+            {"matrix": [8, 0], "x": 130, "y": 48, "flags": 4},
+
+            {"matrix": [9, 0], "x": 132, "y": 64, "flags": 4},
+            {"matrix": [9, 1], "x": 139, "y": 64, "flags": 4},
+            {"x": 148, "y": 64, "flags": 4},
+            {"x": 154, "y": 64, "flags": 2},
+            {"matrix": [9, 2], "x": 163, "y": 64, "flags": 4},
+            {"matrix": [9, 3], "x": 176, "y": 64, "flags": 4},
+            {"matrix": [9, 4], "x": 188, "y": 64, "flags": 4},
+            {"x": 194, "y": 64, "flags": 2},
+            {"matrix": [9, 6], "x": 200, "y": 64, "flags": 4},
+            {"matrix": [9, 7], "x": 212, "y": 64, "flags": 4},
+            {"matrix": [9, 8], "x": 224, "y": 64, "flags": 4},
+            {"x": 224, "y": 52, "flags": 2}
+        ],
+        "max_brightness": 120,
+        "sleep": true,
+        "split_count": [50, 51]
+    },
+    "split": {
+        "bootmagic": {
+            "matrix": [0, 0]
```

Comment:
The intention with this define is to map it to something on the right hand, so i would expect it to be something like `[5, 8]`?

Reply:
Ah yes, will revise, test it, and commit change